internal/io/ioutil/ioutil_test.go

```diff
@@ -18,13 +18,37 @@
 package ioutil
 
 import (
+	"bytes"
+	"os"
 	"reflect"
 	"testing"
 
+	"github.com/vdaas/vald/internal/compress/gob"
 	"github.com/vdaas/vald/internal/errors"
+	"github.com/vdaas/vald/internal/log"
 	"go.uber.org/goleak"
 )
 
+func genStr() []string {
+	str := make([]string, 0)
+	for i := 0; i < 100; i++ {
+		str = append(str, "vdaas.vald.org\n")
+	}
+	return str
+}
+
+func genNonPermittedFile(path string) {
+	fp, err := os.Create(path)
+	if err != nil {
+		log.Error(err)
+	}
+	err = fp.Chmod(0333)
+	if err != nil {
+		log.Error(err)
+	}
+	defer fp.Close()
+}
```

Comment:
should we fatal when error occurs ? 🤔 
```suggestion
func genNonPermittedFile(t *testing.T, path string) {
	t.Helper()
	fp, err := os.Create(path)
	if err != nil {
		t.Fatal(err)
	}
	err = fp.Chmod(0333)
	if err != nil {
		t.Fatal(err)
	}
	defer fp.Close()
}
```

Reply:
maybe, it is more safety 👍